controllers/policy_utils.go

```diff
@@ -82,6 +82,15 @@ func startReconciling(ctx context.Context, client client.Client, reconciler admi
 }
 
 func reconcilePolicy(ctx context.Context, client client.Client, reconciler admission.Reconciler, policy v1alpha2.Policy) (ctrl.Result, error) {
+	apimeta.SetStatusCondition(
+		&policy.GetStatus().Conditions,
+		metav1.Condition{
+			Type:    string(v1alpha2.PolicyActive),
+			Status:  metav1.ConditionFalse,
+			Reason:  "PolicyActive",
+			Message: "The policy webhooks are created",
```

Comment:
```suggestion
			Message: "The policy webhook has not been created",
```

Reply:
The message on this one has to be in negative form given the condition is `False`.